Timestamp: 2021-11-28 21:56:51.439814 | ProcessId: validator_1 | Action: Initial configuration | Object: {'nvalidators': 3, 'nfaulty': 0, 'nclients': 3, 'nclientops': 2, 'sleeptime': 1, 'clienttimeout': 10, 'delta': 2, 'window_size': 5, 'exclude_size': 0, 'failure_config': FailureConfig(failures=[], seed=637713655)} |
Timestamp: 2021-11-28 21:56:59.480068 | ProcessId: validator_1 | Action: Local Timeout occurred | Object: {'round': 1, 'high_qc': None, 'author': 1, 'author_signature': {}} |
Timestamp: 2021-11-28 21:56:59.486403 | ProcessId: validator_1 | Event: Sent | MsgType: Timeout | To: ALL validators | Msg: {'tmo_info': {'round': 1, 'high_qc': None, 'author': 1, 'author_signature': {}}, 'last_round_tc': None, 'high_commit_qc': None}
